modules/graph/loader/basic_ev_fragment_loader_impl.h

```diff
@@ -655,13 +658,28 @@ BasicEVFragmentLoader<OID_T, VID_T, PARTITIONER_T>::constructVerticesImpl(
 
   ObjectID new_vm_id = InvalidObjectID();
   if (vm_id == InvalidObjectID()) {
+    // fill oid array.
+    uint64_t memory_usage = get_rss() / 1024 / 1024;
+    uint64_t max_memory_usage = get_peak_rss() / 1024 / 1024;
+    uint64_t time = GetCurrentTime();
     BasicArrowVertexMapBuilder<internal_oid_t, vid_t> vm_builder(
-        client_, comm_spec_.fnum(), vertex_label_num_, std::move(oid_lists));
+        client_, comm_spec_.fnum(), vertex_label_num_, std::move(oid_lists),
+        use_perfect_hash_);
     // oid_lists.clear();
 
+    // vm_object -> vertex map
     std::shared_ptr<Object> vm_object;
     VY_OK_OR_RAISE(vm_builder.Seal(client_, vm_object));
     new_vm_id = vm_object->id();
+    VLOG(100) << "Vertex map construction time: " << (GetCurrentTime() - time)
+              << "s";
+    uint64_t post_memory_usage = get_rss() / 1024 / 1024;
+    uint64_t post_max_memory_usage = get_peak_rss() / 1024 / 1024;
+    VLOG(100) << "pre memory_usage:" << memory_usage
+              << ", pre max_memory_usage:" << max_memory_usage
+              << ", post memory_usage:" << post_memory_usage
+              << " post max memory usage:" << post_max_memory_usage;
+    VLOG(100) << "Delta memory usage: " << post_memory_usage - memory_usage;
```

Comment:
Don't add logging here, rather, moving those measurements and logs to vm_builder's Seal() method to avoid code duplication.

Reply:
Done.